### PR TITLE
add paper notes on NCNet

### PR DESCRIPTION
**Pull request by FlagArihant2000**

### CODE REVIEW

**Comment by FlagArihant2000** [comment]
@RajGhugare19 @take2rohit 
Kindly have a look at it.
Thanks.

File: deep_learning/README.md [hunk 1]
```diff
@@ -47,7 +48,7 @@
...
 | [D2 Net - A Trainable CNN for Joint Description and Detection of Local Features](https://arxiv.org/abs/1905.03561) (CVPR '19) | [HackMD](https://hackmd.io/@AniketGujarathi/SywvV8iQD) | [Aniket Gujarathi](https://www.linkedin.com/in/aniket-gujarathi/?originalSubdomain=in) | This paper introduces a Deep Learning based approach to solve the problem of local features detection and description using the detect-and-describe approach instead of the traditionally used detect-then-describe approach. |
-
+|[Neighbourhood Consensus Networks](https://arxiv.org/pdf/1810.10510.pdf) (NeurIPS '18)|[HackMD](https://hackmd.io/@GaurArihant/S1NAT2QJO)|[Arihant](https://flagarihant2000.github.io/arihantgaur/), [Saurabh](https://saurabhkemekar.github.io/Saurabh-Kemekar/)|The paper proposes an end - to - end trainable CNN architecture identifying consistent matches by analysing neighbourhood consensus patterns. The paper also demonstrates the use of weak supervision as matching and non - matching pairs, rather than using manual annotations.|
 ### Self Supervised Learning
```
Comment (akshaykvnit) [inline deep_learning/README.md:52]: Leave one line between the table row and title. Not leaving one line causes rendering issue in the corresponding website.
Also, put the papers in reverse chronological order (new papers come first in the table).

**Comment by akshaykvnit** [comment]
@FlagArihant2000 @RajGhugare19 @take2rohit I have added my paper in this PR itself to avoid multiple PRs. Please take care while merging.

**Review by take2rohit — APPROVED** [review]

**Comment by FlagArihant2000** [comment]
Have made the required changes. Another thing I wanted to ask is that I have made notes on another paper (Patch2Pix). Shall I merge them too in this PR?

**Comment by akshaykvnit** [comment]
@FlagArihant2000 Yes, add them in this PR itself.

**Comment by FlagArihant2000** [comment]
@akshaykvnit @take2rohit @RajGhugare19 Any other changes required?

**Pull request merged**